router/batchrouter/batchrouter.go

```diff
@@ -147,14 +147,22 @@ type ErrorResponseT struct {
 	Error string
 }
 
-func updateDestStatusStats(id string, count int, isSuccess bool) {
-	var destStatsD stats.RudderStats
-	if isSuccess {
-		destStatsD = stats.NewBatchDestStat("batch_router.dest_successful_events", stats.CountType, id)
-	} else {
-		destStatsD = stats.NewBatchDestStat("batch_router.dest_failed_attempts", stats.CountType, id)
+func sendDestStatusStats(id string, jobStateCount map[string]int, destType string, isWarehouse bool, parameterFilters []jobsdb.ParameterFilterT) {
+	tags := map[string]string{
+		"module":      "batch_router",
+		"destType":    destType,
+		"isWarehouse": fmt.Sprintf("%t", isWarehouse),
+	}
+	for _, paramFilter := range parameterFilters {
+		tags[fmt.Sprintf("param_%s", paramFilter.Name)] = paramFilter.Value
```

Comment:
The parameter filter also has received_at as a key and the value of received_at timestamp. Timestamps will blow up influx.

Eg
```
{ 
  "source_id": "s1",
  "destination_id": "d1",
  "received_at": "t1"
}
```

Instead of sending all parameter filters, let's send only source and destinations? 
We should use this function for representing any `source` or `destination` in tags.

`misc.GetTagName(destination.Destination.ID, destination.Destination.Name)`

Also the tag names should `source` and `destination` to maintain parity across all metrics.

Reply:
In this context, parameter filters doesn't have received_at. It has only one parameter and that is destId

```
	parameterFilters := []jobsdb.ParameterFilterT{
		{
			Name:     "destination_id",
			Value:    batchJobs.BatchDestination.Destination.ID,
			Optional: false,
		},
	}

```
In any case, I have removed the paramFilter piece and am adding source and destination ids like you suggested. Take a look. Should we add source/dest names here for better readability?